src/constants.ts

```diff
@@ -1,28 +1,28 @@
-import { NodeHelper } from "./helpers/NodeHelper"
-import { EnvHelper } from "./helpers/Environment"
-import ethereum from "./assets/tokens/wETH.svg"
-import arbitrum from "./assets/arbitrum.png"
-import avalanche from "./assets/tokens/AVAX.svg"
+import { NodeHelper } from "./helpers/NodeHelper";
+import { EnvHelper } from "./helpers/Environment";
+import ethereum from "./assets/tokens/wETH.svg";
+import arbitrum from "./assets/arbitrum.png";
+import avalanche from "./assets/tokens/AVAX.svg";
 
-export const THE_GRAPH_URL = "https://api.thegraph.com/subgraphs/name/drondin/olympus-graph"
-export const EPOCH_INTERVAL = 2200
+export const THE_GRAPH_URL = "https://api.thegraph.com/subgraphs/name/drondin/olympus-graph";
+export const EPOCH_INTERVAL = 2200;
 
 // NOTE could get this from an outside source since it changes slightly over time
-export const BLOCK_RATE_SECONDS = 13.14
+export const BLOCK_RATE_SECONDS = 13.14;
```

Comment:
@0xtaichi following up on my last comment, Fantom's block time will not be a constant. What does the frontend use these 2 variables for? We will have to work around them.